scripts/generate-federate-match.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+shopt -s inherit_errexit
+
+! [ -x "$(command -v jq)" ] && echo 'jq not installed, the script requires it.' && exit 1
```

Comment:
Nit: might be nicer to echo all missing requirements in one go, so they can be installed in one go.
Or just point to the README? 🤔